### DISCUSSION TITLE
how to put the dynamic url variable of a button?

### DISCUSSION THREAD

**Discussion by vandrovisky** [netflie/whatsapp-cloud-api#95] [General]
hello everybody. How can I send the variable {{1}} from a dynamic url using Send Template?
![Captura de tela de 2023-03-09 12-34-03](https://user-images.githubusercontent.com/88595552/224073888-d2fd16d8-5fa1-420c-8461-0ab752b475df.png)

**Comment by Jhamnerx** [comment]
![image](https://user-images.githubusercontent.com/30221273/224077591-fbe9dd12-3de3-4cec-92ef-1786405e8e36.png)
Hello, just leave it as you have it.
just send the parameter and it will be added to the end of your dynamic url.

**It is not necessary to put {{1}}**

**Reply by vandrovisky** [reply]
Thanks for your help identifying that the sub_url was missing from my code through your example!
Thank you very much!!
